### ISSUE TITLE
Missing dependency: php-xml

### ISSUE THREAD

**Issue by verement** [humhub/humhub#3713]
Hello,

Attempting to export the users list to xlsx format fails unless a PHP XML extension is installed (e.g. Debian package php-xml). Such an extension is not listed among the [installation requirements](http://docs.humhub.org/admin-requirements.html).

### What steps will reproduce the problem?
Go to Administration > Users and click on Export > xlsx. If the required PHP extension is not installed, an error occurs.

### What is the expected result?
The export should happen correctly, or the dependency should have been documented at the time of installation.

### What do you get instead?
An error is logged.

### Additional info
It would be great to have this also checked for automatically among the software prerequisites checklist.

| Q                | A
| ---------------- | ---
| HumHub version   | 1.3.17
| PHP version      | 7.3.11
| Operating system | Debian 10.1

> Note: Also provide the full error stacks from `Administration --> Information --> Logging` and errors from your browser console if possible.
```
Error: Class 'XMLWriter' not found in /var/www/humhub/protected/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Shared/XMLWriter.php:5
Stack trace:
#0 /var/www/humhub/protected/vendor/composer/ClassLoader.php(444): include()
#1 /var/www/humhub/protected/vendor/composer/ClassLoader.php(322): Composer\Autoload\includeFile('/var/www/humhub…')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('PhpOffice\\PhpSp…')
#3 /var/www/humhub/protected/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Writer/Xlsx/ContentTypes.php(30): spl_autoload_call('PhpOffice\\PhpSp…')
#4 /var/www/humhub/protected/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Writer/Xlsx.php(223): PhpOffice\PhpSpreadsheet\Writer\Xlsx\ContentTypes->writeContentTypes(Object(PhpOffice\PhpSpreadsheet\Spreadsheet), false)
#5 /var/www/humhub/protected/humhub/components/export/ExportResult.php(124): PhpOffice\PhpSpreadsheet\Writer\Xlsx->save('/var/www/humhub…')
#6 /var/www/humhub/protected/humhub/components/export/ExportResult.php(138): humhub\components\export\ExportResult->saveAs('/var/www/humhub…')
#7 /var/www/humhub/protected/humhub/modules/admin/controllers/UserController.php(348): humhub\components\export\ExportResult->send()
#8 [internal function]: humhub\modules\admin\controllers\UserController->actionExport('xlsx')
#9 /var/www/humhub/protected/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#10 /var/www/humhub/protected/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#11 /var/www/humhub/protected/vendor/yiisoft/yii2/base/Module.php(528): yii\base\Controller->runAction('export', Array)
#12 /var/www/humhub/protected/vendor/yiisoft/yii2/web/Application.php(103): yii\base\Module->runAction('admin/user/expo...', Array)
#13 /var/www/humhub/protected/vendor/yiisoft/yii2/base/Application.php(386): yii\web\Application->handleRequest(Object(humhub\components\Request))
#14 /var/www/humhub/index.php(25): yii\base\Application->run()
#15 {main}
```

> Note: Please use the [community](https://community.humhub.com) site for any questions on customizing HumHub.

**Comment by Ace2213** [comment]
It's listed in /admin/information/prerequisites. It just needs to be added to the documentation.

![image](https://user-images.githubusercontent.com/12226710/68038615-20b64700-fca9-11e9-8819-88e208651626.png)

**Comment by verement** [comment]
> It's listed in /admin/information/prerequisites. It just needs to be added to the documentation.

Interesting… It's not listed among the prerequisites on my installation. What version of HumHub is that from?

**Comment by Ace2213** [comment]
1.4 beta. Perhaps it was added in the new version.

**Comment by luke-** [comment]
Indeed this is fixed in 1.4+:
https://github.com/humhub/humhub/blob/develop/protected/humhub/libs/SelfTest.php#L178-L192
